AndroidSDKCore/src/main/java/com/leanplum/Leanplum.java

```diff
@@ -2080,6 +2084,24 @@ public void error(Exception e) {
     }
   }
 
+  /**
+   * Forces downloading of inbox messages from the server. After messages are downloaded the
+   * appropriate callbacks will fire.
+   */
+  public static void downloadInboxMessages() {
+    downloadInboxMessages(null);
+  }
+
+  /**
+   * Forces downloading of inbox messages from the server. After messages are downloaded the
+   * appropriate callbacks will fire.
+   *
+   * @param callback The callback to invoke when messages are downloaded.
+   */
+  public static void downloadInboxMessages(@Nullable InboxSyncedCallback callback) {
```

Comment:
Should we move those 2 methods to be accessible only from inbox() instance ?

Reply:
This is a good question, I thought that it would be better to add them in Leanplum class, but now I see that it will be better to move them to LeanplumInbox, because there already are several public methods for registering the callbacks.
I will update the PR.